plugins/rollbar_dashboards/config/routes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
where is that used ?

Reply:
Remanent of a previous version of this feature 😅